pkg/microservice/warpdrive/core/service/taskcontroller/task_handler.go

```diff
@@ -282,9 +282,8 @@ func (h *ExecHandler) runStage(stagePosition int, stage *task.Stage) {
 	// Default worker concurrency is 1, run tasks sequentially
 	var workerConcurrency = 1
 	if runParallel {
-		// MaxWorkerInParallel is 5 for now
-		if len(stage.SubTasks) > maxWorkerInParallel {
```

Comment:
Do we need to delete the `pkg/microservice/warpdrive/core/service/taskcontroller/const.go` file if `maxWorkerInParallel` is no longer used?